adapters/bevy/shared/src/flag.rs

```diff
@@ -3,10 +3,9 @@ pub struct Flag {
 }
 
 impl Flag {
+    #![allow(clippy::new_without_default)]
```

Comment:
But why just not use default trait?

Reply:
The warning won't be triggered in case of only `default`.

Reply:
Well for some reason even when I was using the default trait, it built on my machine but then wouldn't build in the CI ! The compiler said "you need to implement Default" when it clearly WAS implemented. So moving to a plain new() method got it to compile, but then Clippy complains that it should be default :man_facepalming: ...

Reply:
It worked before, I already fixed this warning by replacing `new` with `Default` macro.

Reply:
You can see in the PR diff that you removed `Default` macro.

Reply:
Sorry sorry, it wasn't the CI that wouldn't compile, it was `cargo publish` that for some reason complained about Default not being implemented.

You know what.. I'm going to revert this back and leave fixing this properly for later :P

Reply:
Oh, that strange :)